services/web-app/pages/contributing/schema.mdx

```diff
@@ -192,17 +212,27 @@ asset key to override the inherited default.
 Assets are reusable units of work that are used in products and digital experiences. Every asset
 belongs to a library.
 
-- For each asset in a library, create a `carbon.yml` metadata file in the same directory as the
-  asset's source and place the asset details inside of the `assets` object with the asset `id` as
-  the key.
-- For directories that contain multiple assets, just add each asset as a singular entry to the
-  `assets` object with the asset `id` as the key.
+<AnchorLinks small>
 
-**Example**
+<AnchorLink>Asset keys</AnchorLink>
```

Comment:
It's unusual to have this second layer of anchor links-- for this version I think it's a good start. Let's consider addressing deep navigation later.